tests/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -231,6 +233,240 @@ TEST (nnstreamer_capi_valve, test01)
   g_free (pipeline);
 }
 
+/**
+ * @brief A tensor-sink callback for sink handle in a pipeline
+ */
+static void nns_sink_callback_dm01 (const char *buf[], const size_t size[],
+    const GstTensorsInfo *tensorsinfo, void *pdata)
+{
+  gchar *filepath = (gchar *) pdata;
+  FILE *fp = g_fopen (filepath, "a");
+  int i, num = tensorsinfo->num_tensors;
+
+  for (i = 0; i < num; i++) {
+    fwrite (buf[i], size[i], 1, fp);
+  }
+
+  fclose (fp);
+}
+
+/**
+ * @brief compare the two files.
+ */
+static int file_cmp (const gchar *f1, const gchar *f2)
+{
+  gboolean r;
+  gchar *content1, *content2;
+  gsize len1, len2;
+  int cmp = 0;
+
+  r = g_file_get_contents (f1, &content1, &len1, NULL);
+  if (r != TRUE)
+    return -1;
+
+  r = g_file_get_contents (f2, &content2, &len2, NULL);
+  if (r != TRUE) {
+    g_free (content1);
+    return -2;
+  }
+
+  if (len1 == len2) {
+    cmp = memcmp (content1, content2, len1);
+  } else {
+    cmp = 1;
+  }
+
+  g_free (content1);
+  g_free (content2);
+
+  return cmp;
+}
+
+/**
+ * @brief Test NNStreamer pipeline sink
+ */
+TEST (nnstreamer_capi_sink, dummy_01)
+{
+  const gchar *_tmpdir = g_get_tmp_dir ();
+  const gchar *_dirname = "nns-tizen-XXXXXX";
+  gchar *fullpath = g_build_path ("/", _tmpdir, _dirname, NULL);
+  gchar *dir = g_mkdtemp ((gchar *) fullpath);
+  gchar *file1 = g_build_path ("/", dir, "original", NULL);
+  gchar *file2 = g_build_path ("/", dir, "sink", NULL);
+  gchar *pipeline = g_strdup_printf ("videotestsrc num-buffers=3 ! videoconvert ! video/x-raw,format=BGRx,width=64,height=48,famerate=60/1 ! tee name=t t. ! queue ! filesink location=\"%s\"  t. ! queue ! tensor_converter ! tensor_sink name=sinkx",
+      file1);
+  nns_pipeline_h handle;
+  nns_pipeline_state_e state;
+  nns_sink_h sinkhandle;
+  int status = nns_pipeline_construct (pipeline, &handle);
+  EXPECT_EQ (status, NNS_ERROR_NONE);
+
+  EXPECT_TRUE (dir != NULL);
```

Comment:
Insert this check after line 293
Also, should be ASSERT() as EXPECT will continue on failure.

Reply:
Updated.